datasets/imdb.py

```diff
@@ -40,9 +40,9 @@ def sort_key(ex):
         return len(ex.text)
 
     @classmethod
-    def splits(cls, path, train=os.path.join('IMDB', 'data', 'imdb_train.tsv'),
-               validation=os.path.join('IMDB', 'data', 'imdb_validation.tsv'),
-               test=os.path.join('IMDB', 'data', 'imdb_test.tsv'), **kwargs):
+    def splits(cls, path, train=os.path.join('IMDB', 'train.tsv'),
```

Comment:
This change doesn't break previous code, right?

Reply:
It doesn't great previous code. Directory structure is different (in a way simpler) in Hedwig when compared to Castor.